packages/ove-app-images/README.md

```diff
@@ -0,0 +1,53 @@
+# Images App
+
+This app supports visualisation of images using the OVE framework. It is based on [OpenSeadragon](https://openseadragon.github.io/) and supports high resolution zoomable images.
```

Comment:
Clarify that it supports both 'normal' images, and sources of image tiles for high-resolution zoomable images.